tests/test_domains.py

```diff
@@ -67,29 +66,27 @@ def _create_class(data):
         self.assertIsInstance(cls, dotnet.DotNetClass)
         self.assertDictEqual(cls.item_map, {})
 
+    def _mock_find(self, patterns, **kwargs):
+        return {'items': ['foo', 'bar']}
+
+    def _mock_read(self, path):
+        return {'items': [{'id': 'Foo.Bar', 'name': 'Foo', 'type': 'property'},
+                          {'id': 'Foo.Bar2', 'name': 'Bar', 'type': 'property'}],
+                'id': 'Foo.Bar', 'type': 'Class', 'summary': path}
```

Comment:
Should probably just define these methods as standard mock objects instead of moving them out of the test scope they are established for